src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.TargetFrameworkInference.targets

```diff
@@ -154,8 +154,8 @@ Copyright (c) .NET Foundation. All rights reserved.
       <TFTelemetry Include="PublishSelfContained" Value="$(PublishSelfContained)" />
       <TFTelemetry Include="PublishReadyToRun" Value="$(PublishReadyToRun)" />
       <TFTelemetry Include="PublishReadyToRunComposite" Value="$(PublishReadyToRunComposite)" />
-      <TFTelemetry Include="PublishProtocol" Value="$(PublishProtocol)" />    
-      <TFTelemetry Include="Configuration" Value="$(Configuration)" />
+      <TFTelemetry Include="PublishProtocol" Value="$(PublishProtocol)" />
+      <TFTelemetry Include="Configuration" Value="$(Configuration)" Hash="true" />
```

Comment:
After talking to Daniel, we should:
* change the default of AllowEmptyTelemetry so that values are hashed by default and require explicit opt out (so we can be safe)
* audit and annotate all of the existing usage of AllowEmptyTelemetry to align with this new default